vars/scenarioUI5_SAP_CP.groovy

```diff
@@ -0,0 +1,60 @@
+import static com.sap.piper.Prerequisites.checkScript
```

Comment:
Not a big fan of the caps and underscores mix in the naming. The names I'm aware of are using strict CamelCase, I would stick with that: Suggestions: `scenarioUi5onCloudPlatform`, `scenarioFioriOnCloudPlatform` or `scenarioMtaOnCloudPlatform`. What do you think?

Reply:
changed to scenarioFioriOnCloudPlatform